edc-tests/e2e-tests/src/test/resources/summary-vc.json

```diff
@@ -0,0 +1,38 @@
+{
+  "@context": [
+    "https://www.w3.org/2018/credentials/v1",
+    "https://www.w3.org/2018/credentials/examples/v1"
+  ],
+  "id": "urn:uuid:12345678-1234-1234-1234-123456789abc",
+  "type": [
+    "VerifiableCredential",
+    "SummaryCredential"
+  ],
+  "issuer": "did:web:a016-203-129-213-99.ngrok-free.app:BPNL000000000000",
+  "issuanceDate": "2023-06-02T12:00:00Z",
+  "expirationDate": "2022-06-16T18:56:59Z",
```

Comment:
How can be the expiration date before the issuanceDate?

Reply:
@florianrusch-zf  this is just for testing